assets/js/modules/analytics-4/components/audience-segmentation/settings/SettingsCardVisitorGroups/index.js

```diff
@@ -71,8 +78,12 @@ export default function SettingsCardVisitorGroups() {
 				<Grid>
 					<Row>
 						<Cell size={ 12 }>
-							{ ! configuredAudiences?.length && <SetupCTA /> }
-							{ !! configuredAudiences?.length && (
+							{ ! configuredAudiences &&
+								audienceSegmentationSetupCompletedBy ===
+									null && <SetupCTA /> }
+							{ ( configuredAudiences ||
+								audienceSegmentationSetupCompletedBy !==
+									null ) && (
```

Comment:
It would be easier to understand at a glance that these are mutually exclusive conditions if we extract the first to a variable and then invert it here, i.e.

```js
const showSetupCTA = ! configuredAudiences && audienceSegmentationSetupCompletedBy === null;
```

```suggestion
							{ showSetupCTA && <SetupCTA /> }
							{ ! showSetupCTA && (
```